src/components/transactions/modals/ClaimWRBTCModal.vue

```diff
@@ -98,7 +98,7 @@
 <script>
 import { store } from '@/store'
 import { CLAIM_TYPES } from '@/constants/claimTypes'
-import { findNetworkByChainId } from '@/constants/networks'
+import { findNetworkByChainId, findNetworkById } from '@/constants/networks'
```

Comment:
what's the difference between these 2 functions?

Reply:
Removed